### ISSUE TITLE
Error Running Container - Tag 2.5-ss

### ISSUE THREAD

**Issue by SC7639** [jacobalberty/firebird-docker#16]
When I start the container it fails instantly with the logged line: ```/usr/local/firebird/docker-entrypoint.sh: line 19: kill: (23187) - No such process```

**Comment by jacobalberty** [comment]
What command line are you using to launch docker? Are you using an existing security2.fdb or letting the image create its own?

**Comment by SC7639** [comment]
I'm using docker-compose with a volume that stores the database.

**Comment by jacobalberty** [comment]
Can you post your docker-compose.yml in here then?

**Comment by juandows** [comment]
I have similar problem. Sometimes container firebird exit: /usr/local/firebird/docker-entrypoint.sh: line 19: kill: (13) - No such process

Here is my docker-compose.yml:

version: '3'

services:
  web:
    build: .
    #command: python manage.py runserver 0.0.0.0:8000
    volumes:
      #- ./www:/var/www/html
      - ./www:/home/www
    ports:
      - "8000:80"
    links:
      - sessions
    extra_hosts:
      - "barcodes1.redyser.com:192.168.90.75"
      - "mongo1.redyser.com:192.168.91.143"
      - "barcodes2.redyser.com:192.168.91.124"
      - "mongo2.redyser.com:192.168.90.118"
  sessions:
    image: jacobalberty/firebird:2.5-ss
    environment:
    - ISC_PASSWORD=masterkey
    volumes:
      - ./ibase:/databases/
    ports:
      - "3050:3050"


¿it is incorrect?

**Comment by jacobalberty** [comment]
I've been unable to reproduce it, but I think I may have a fix. There seems to be some other factor going on in your setup so I can't confirm this fix myself. Try the branch 2.5-ss-test. It's building on the docker hub now so give it about 20 minutes to finish. Please report back. If it fixes your issue then I will merge it into the main 2.5 branches and retag the most recent 2.5 releases.

**Comment by juandows** [comment]
Sorry, with branch 2.5-ss-test no fail. I execute five times with
2.5-ss-test and all ok . With 2.5-sc is ok, but with 2.5-sc "exited with
code 1"

I changed 2.5-ss -> 2.5-sc and all is ok.

If i can help you, only said me.

Docker version 17.06.0-ce, build 02c1d87
docker-compose version 1.13.0, build 1719ceb

**Comment by jacobalberty** [comment]
So 2.5-ss-test works fine?

and 2.5-sc works fine too?

Only 2.5-ss fails?

**Comment by juandows** [comment]
Yes

**Comment by jacobalberty** [comment]
This should be fixed as of 5a9ec21 which is building right now on the docker hub under both 2.5-ss and 2.5.7-ss tags on the hub.


I'll leave the issue open another week or two in case someone else is still having this issue or finds something wrong with the fix.

**Comment by LVR87** [comment]
Hi, I have the same problem when try to use docker-compose with 2.5-ss. 

```
version: "3.9"
services:
  firebird-2.5:
    image: "jacobalberty/firebird:2.5-ss"
    container_name: "firebird-2.5-ss"
    restart: always
    ports:
      - "3051:3050"
    volumes:
      - ./firebird-2.5:/firebird
    environment:
      - ISC_USER=sysdba
      - ISC_PASSWORD=masterkey 
 ```
 
 after run command `docker-compose up --build`, I got:

 ```
 [+] Running 2/1
 - Network firebird_default   Created                                                                                                                                            0.7s
 - Container firebird-2.5-ss  Created                                                                                                                                            0.1s
Attaching to firebird-2.5-ss
firebird-2.5-ss exited with code 124
firebird-2.5-ss exited with code 0
firebird-2.5-ss exited with code 0
firebird-2.5-ss exited with code 0 
```
